Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -525,10 +523,24 @@ bool VolumeVerifier::CheckPartition(const Partition& partition)
 
   if (!m_is_datel)
   {
-    CheckCorrectlySigned(
-        partition,
-        StringFromFormat(Common::GetStringT("The %s partition is not correctly signed.").c_str(),
-                         name.c_str()));
+    IOS::HLE::Kernel ios;
+    const auto es = ios.GetES();
+    const std::vector<u8>& cert_chain = m_volume.GetCertificateChain(partition);
+
+    if (IOS::HLE::IPC_SUCCESS !=
+            es->VerifyContainer(IOS::HLE::Device::ES::VerifyContainerType::Ticket,
+                                IOS::HLE::Device::ES::VerifyMode::DoNotUpdateCertStore,
+                                m_volume.GetTicket(partition), cert_chain) ||
+        IOS::HLE::IPC_SUCCESS !=
+            es->VerifyContainer(IOS::HLE::Device::ES::VerifyContainerType::TMD,
+                                IOS::HLE::Device::ES::VerifyMode::DoNotUpdateCertStore,
+                                m_volume.GetTMD(partition), cert_chain))
+    {
+      AddProblem(Severity::Low,
+                 std::move(StringFromFormat(
```

Comment:
Do you need this `std::move` here? I realize the original `CheckCorrectlySigned` method had it, but the argument would be to either use it consistently in all places or not at all (if it is elided anyways).

The alignment `if` right before this doesn't have it for example, nor do the newly added checks for alignment; but then we have others like the integrity check `if` right afterwards that does.

Reply:
This `std::move` is unnecessary. I just hadn't noticed it was there.

The integrity check uses `std::move` because the string is stored into a variable, which was done because the code otherwise would be formatted like this:

```
AddProblem(
    Severity::Low,
    StringFromFormat(
        Common::GetStringT("The H3 hash table for the %s partition is not correct.").c_str(),
        name.c_str()));
```